mathics/builtin/atomic/numbers.py

```diff
@@ -960,6 +961,7 @@ class MinPrecision(Builtin):
 
 class NumericQ(Builtin):
     """
+
```

Comment:
Why add a line here and remove it from below?

Reply:
Let's get rid of these black lines that have been added. I don't see that this is doing anything.